bedrock/legal/forms.py

```diff
@@ -97,9 +97,7 @@ class FraudReportForm(forms.Form):
             }
         )
     )
-    input_attachment = forms.FileField(
-        required=False,
-    )
+    input_attachment = forms.ImageField(required=False)
```

Comment:
From the docs, it appears this auto-validates format to ensure the file is of a type understood by Pillow. Is that right?

Reply:
That is my understanding as well. And there is a test that tries to upload a text file and fails.

Reply:
I guess my question is: will this allow things like .pdf and .tiff? Do we want to allow those kinds of images? Should we force a restriction to the three "standards": .png, .gif, and .jpg?

Reply:
Might allow .tiff, but doesn't allow .pdf.

Reply:
These are the supported formats http://pillow.readthedocs.io/en/latest/handbook/image-file-formats.html

Reply:
I believe we could filter on image type if we need to.

Reply:
That's quite the list! I think we're pretty well covered. We can limit at a later date if needed.